docs/en/credential-issuance-endpoint.rst

```diff
@@ -831,7 +837,7 @@ The Credential Response contains the following parameters:
     - **Description**
     - **Reference**
   * - **credentials**
-    - REQUIRED if ``lead_time`` and ``transaction_id`` are not present, otherwise it MUST NOT be present. It contains the following parameters:
+    - REQUIRED if ``lead_time`` and ``transaction_id`` are not present, otherwise it MUST NOT be present. It is an array of one or more issued Credentials. The elements of the array MUST be objects. The number of elements in the credentials array matches the number of keys that the Wallet Instance has provided either via the ``proof`` or ``proofs`` parameter of the Credential Request. It contains the following parameters:
```

Comment:
```suggestion
    - REQUIRED if ``lead_time`` and ``transaction_id`` are not present, otherwise it MUST NOT be present. It is an array of one or more issued Credentials. The number of elements in the Credentials array matches the number of keys that the Wallet Instance has provided either via the ``proof`` or ``proofs`` parameter of the Credential Request. The array MUST contain JSON objects, where each object MUST have exactly one member with the key ``credential``. The value of the ``credential`` member MUST be a string containing the encoded Credential, as further described below:
```